### PR TITLE
manifest: update zephyr with qdec tests

### PR DESCRIPTION
**Pull request by nordic-piks**
QDEC tests.

### CODE REVIEW

**Review by nordic-bami — APPROVED** [review]

**Review by nordic-segl — APPROVED** [review]

**Comment by nordic-piks** [comment]
Rebased after NCS 2.7

File: west.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: ded7e9fcce1ea756b8f26e652248d026ed732be7
+      revision: pull/1871/head
```
Comment (anangl) [inline west.yml:66]: Please update with SHA: `6e6f706860ed490ae6b0a27411fa4a87db3a21a8`.

Comment (nordic-piks) [reply west.yml]: Done, thanks :)

**Review by anangl — APPROVED** [review]

**Comment by nordic-piks** [comment]
Zephyr already update, this PR is not needed.

**Pull request closed without merging**